### PR TITLE
Apply campus code fixup to course data

### PR DESCRIPTION
**Pull request by timabell**
This was causing the in-memory join to fail to find due to the mismatch
in values.

### Context
Fire 🔥 

### Guidance to review

This is the join: https://github.com/DFE-Digital/manage-courses-api/blob/c7be25bc6b9b10ddd6b0e510311166d3c574659b/src/ManageCourses.UcasCourseImporter/xls-reader/XlsReader.cs#L72

### CODE REVIEW

**Comment by timabell** [comment]
[missing courses](https://trello.com/c/sQHZyKVX/1026-missing-courses)

File: src/ManageCourses.UcasCourseImporter/xls-reader/XlsReader.cs [hunk 1]
```diff
@@ -345,5 +345,9 @@ public List<UcasNoteText> ReadNoteText(string folder)
 
             return result;
         }
+
+        private string CampusCodeFixer(string code){
+            return code.Replace(" ", "-").Trim();
```
Comment (defong) [inline src/ManageCourses.UcasCourseImporter/xls-reader/XlsReader.cs]: return code.Trim().Replace(" ", "-");

Comment (defong) [reply src/ManageCourses.UcasCourseImporter/xls-reader/XlsReader.cs]: return code.Trim().Replace("", "-");

Comment (defong) [reply src/ManageCourses.UcasCourseImporter/xls-reader/XlsReader.cs]: one or both of those

Comment (timabell) [reply src/ManageCourses.UcasCourseImporter/xls-reader/XlsReader.cs]: optimise later

Comment (defong) [reply src/ManageCourses.UcasCourseImporter/xls-reader/XlsReader.cs]: lets hope `code` != buggy 

we should be a little bit more careful and null check

**Pull request merged**